src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -27,6 +28,11 @@ public enum ODataLibraryCompatibility
         /// <summary>
         /// The latest version of the library.
         /// </summary>
-        Latest = int.MaxValue
+        Latest = int.MaxValue,
+
+        /// <summary>
+        /// Disables the CSDL Scale and SRID variable value from being written as variable to Variable.
+        /// </summary>
+        UseLegacyVariableCasing = 01

```

Comment:
```suggestion
        UseLegacyScaleVariableCasing = 1
```
Or if bit-shifting would easier to visualize:
```
UseLegacyScaleVariableCasing = 1 << 0 // Equivalent to 1
Foo = 1 << 1 // Equivalent to 2
Bar = 1 << 2 // Equivalent to 4
SoOnAndSoForth = 1 << N // where N < 63

Reply:
Since `SRID` and `Scale` both use `variable`, I don't think we need to have this flag as `UseLegacyScaleVariableCasing`. I think by having it as `UseLegacyVariableCasing` it can be shared by both.